php/Terminus/Models/Environment.php

```diff
@@ -384,6 +384,22 @@ public function domain() {
   }
 
   /**
+   * Get the code log (commits)
+   *
+   * @return array
+   */
+  public function getCodeLog() {
+    $path     = sprintf('environments/%s/code-log', $this->get('id'));
+    $response = $this->request->request(
+      'sites',
+      $this->site->get('id'),
+      $path,
+      'GET'
+    );
+    return $response['data'];
```

Comment:
Can we write this as model and collection? Or just write it as "hasCommits" if our only usage right now is to verify that `count(commits) > 0`.